returnn/tf/layers/basic.py

```diff
@@ -6114,14 +6116,21 @@ def __init__(
         filter=None,
         filter_perm=None,
         bias=None,
+        use_time_mask=None,
```

Comment:
Why `None` as the default value? I thought this is supposed to be a bool?

Reply:
```suggestion
        use_time_mask=False,
```

Reply:
i looked at other layers which already had this option (e.g. ReduceLayer) and there the default was also `None`

Reply:
But there the situation is different. First, they accept the type `Optional[bool]`, which allows None. Sometimes in other cases you might also see `Union[bool, NotSpecified]`. Basically, `None` is treated like `NotSpecified`, meaning that it is not specified. Then, in the code, the default value is not simply a True or False (otherwise this would not be needed), but it depends on the behavior version, and maybe other things.

However, here in your case, we simply have one default value (at least for now), which is simply `False`. None of the logic is needed then. You should only allow the type `bool`.

Reply:
Okay, yes, that makes sense.

Reply:
But you did not fix this yet?